siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py

```diff
@@ -264,6 +264,21 @@ def exec_functions(self, device_ids, function_id, args=None):
         self._queue.append(operation)
         return True
 
+    def update_parameters(self, device_ids):
+        """Update device parameters."""
+        # if in SOFBMode on, do not accept comm. commands
+        if self._sofb_mode:
```

Comment:
the last commit got me confused about the motivation of this PV. I thought it was created to update the parameters of the Power Supply when it were in SOFB mode, because otherwise they would always be updated automatically. But it seems this is not the case.

Reply:
The parameters we are handling here are those related to "-Cte" PVs that we show in the PS Parameters auxiliary window. The reading of those parameters is an expensive task (it takes ~0.7s per power supply) and is only performed in IOC initialization. The idea here is to make it possible to update these PVs "on the fly" during maintenance activities, such as in the following weeks.

Reply:
Sorry guys for not adding a description for this PR